docs/topics/whatsnew2020.md

```diff
@@ -584,6 +584,27 @@ By default, this property is set to `false` and the `apiElements` variant in Gra
 We would appreciate your feedback on this new approach. Have you noticed any performance improvements while using it?
 Let us know by adding a comment in [YouTrack](https://youtrack.jetbrains.com/issue/KT-61861/Gradle-Kotlin-compilations-depend-on-packed-artifacts).
 
+### Aligned dependency behavior of Kotlin Gradle plugin with java-test-fixtures plugin
+
+Prior to Kotlin 2.0.20, if you used the [`java-test-fixtures` plugin](https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures)
+in your project, there was a difference between Gradle and the Kotlin Gradle plugin in how dependencies were propagated. 
+
+The Kotlin Gradle plugin propagated dependencies:
+
+* From the `java-test-fixtures` plugin's `implementation` and `api` configurations to the `test` sources compilation classpath.
```

Comment:
```suggestion
* From the `java-test-fixtures` plugin's `implementation` and `api` configurations to the `test` source set compilation classpath.
```

Reply:
Here and below, in our documentation, we generally refer to such dependency types as `implementation` and `api` as "scopes", not "configurations"

Reply:
Here we refer them just as "dependency types": https://kotlinlang.org/docs/gradle-configure-project.html#dependency-types

Gradle calls them configurations: https://docs.gradle.org/current/userguide/dependency_configurations.html#sec:what-are-dependency-configurations